tests/e2e/framework/util.go

```diff
@@ -161,3 +162,23 @@ func WaitForNamespacesDeleted(ctx context.Context, c clientset.Interface, namesp
 			return true, nil
 		})
 }
+
+// helmDebugLog prints the debug log of the helm client into a file in the test directory.
+func helmDebugLog(format string, v ...interface{}) {
+	if _, err := os.Stat("e2e-helm.log"); os.IsNotExist(err) {
+		_, err := os.Create("e2e-helm.log")
+		ExpectNoError(err)
+	}
+
+	outFile, err := os.OpenFile("e2e-helm.log", os.O_APPEND|os.O_WRONLY, 0644)
+	ExpectNoError(err)
+
+	defer outFile.Close()
+
+	// if formart doesn't end with newline, add it
+	if !strings.HasSuffix(format, "\n") {
+		format += "\n"
+	}
+
+	fmt.Fprintf(outFile, format, v...)
```

Comment:
By default the helm logger uses `log.Printf(format, v...)`. Does it make sense to use the equivalent here?

Reply:
sure, makes sense

Reply:
I mean do we want the timestamp and other information?